tests/resources/lsp_transpiler/lsp_server.py

```diff
@@ -0,0 +1,151 @@
+import os
+import sys
+from collections.abc import Sequence
+from typing import Any, Literal
+from uuid import uuid4
+
+import attrs
+
```

Comment:
@ericvergnaud  Can you leave docs or notes on why this is needed for unit tests and why is this implementation under resources.

Reply:
done